include/model_mgmt.h

```diff
@@ -74,7 +74,7 @@ class i_model
 {
 public:
   virtual int update(const model_data& data, bool& model_ready, api_status* status = nullptr) = 0;
-  virtual int choose_rank(uint64_t rnd_seed, string_view features, std::vector<int>& action_ids,
+  virtual int choose_rank(const char* event_id, uint64_t rnd_seed, string_view features, std::vector<int>& action_ids,
```

Comment:
Should be a string_view instead of const char*

Reply:
@jackgerrits this is consistent with event_id's in other function signatures, the PR where we changed all event_id's to string_views has yet to be merged

Reply:
Ah okay, I thought it had been merged.